### DISCUSSION TITLE
v2.0.0-beta.4

### DISCUSSION THREAD

**Discussion by shuding** [shuding/nextra#270] [General]
## What's Changed
* Fix `filter of undefined` caused by MDX tree being empty by @shuding in https://github.com/shuding/nextra/pull/269

**Full Changelog**: https://github.com/shuding/nextra/compare/v2.0.0-beta.3...v2.0.0-beta.4

<hr /><em>This discussion was created from the release <a href='https://github.com/shuding/nextra/releases/tag/v2.0.0-beta.4'>v2.0.0-beta.4</a>.</em>

**Comment by pomber** [comment]
Hi, I'm giving v2 a try. Is it still possible to pass an `mdxOptions` object?

This used to work in v1, but looks like it's ignored in v2 (beta.5):

```js
// next.config.js
function remarkFoo() {
  return (tree, file) => {
    console.log("foo");
  };
}

const withNextra = require("nextra")({
  theme: "nextra-theme-docs",
  themeConfig: "./theme.config.js",
  mdxOptions: {
    remarkPlugins: [remarkFoo],
  },
});
module.exports = withNextra();

```
